src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -125,11 +125,12 @@ class TestLoadManagerBase {
     }
   }
 
+  std::shared_ptr<cb::MockClientStats> stats_;
```

Comment:
why was this moved?

Reply:
To access it in the test.

Reply:
just depends on how we want it to be used. I think there is a GetStats() function? If it was a data-only class this would be fine. Now that we have added an input control to it, maybe it needs to be public